### DISCUSSION TITLE
external task sensor seems to not work

### DISCUSSION THREAD

**Discussion by 2myeonggyu** [apache/airflow#23750] [General]
### Apache Airflow version

2.3.0 (latest released)

### What happened

Hi there.
In my airflow tasks, I used external task sensor operator to complete dependent tasks.
It was perfectly work until updating airflow version from v2.0.1 to v2.3.0.
But now after updating, the operation is not working(auto failure) with no log...
Is there a bug? or did I miss someting?

Please help.. 

### Operating System

ubuntu 20.04.4

### Deployment

Docker-Compose

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by potiuk** [comment]
It's not a known issue. But without any detailed description and reproducibilility we cannot do much.  ANy more details and analysis of logs are necessary to do any help with your diagnosis. There might be a number of reasons why things woudl not work, and your guess - witthout any logs indicating kind of problems, is not going to help us to help you. You need to provide more details, look at your logs (including your docker-compose/deployment logs) and find any indication of the problem.
